subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -441,7 +441,8 @@ static struct {
 #define LINK_RX_POOL_SIZE                                                      \
 	(sizeof(memq_link_t) *                                                 \
 	 (RX_CNT + 2 + BT_CTLR_MAX_CONN + BT_CTLR_ADV_SET +                    \
-	  (BT_CTLR_SCAN_SYNC_SET * 2) + (BT_CTLR_SCAN_SYNC_ISO_SET * 2) +      \
+	  (BT_CTLR_ADV_ISO_SET * 2) + (BT_CTLR_SCAN_SYNC_SET * 2) +            \
```

Comment:
Does `BT_CTLR_ADV_ISO_SET` refer to any broadcast ISO, or just the broadcaster or receiver?

Reply:
It is the number of supported simultaneous Broadcast ISO Sets.

Reply:
And those sets are also used for the ISO Sync Receiver?

Reply:
No, `BT_CTLR_SCAN_SYNC_ISO_SET` is for ISO Sync Receiver.

Reply:
Should it not have been `BT_CTLR_SCAN_SYNC_ISO_SET` instead of `BT_CTLR_ADV_ISO_SET` for this change then, since it is for the `LINK_RX_POOL_SIZE`? Unless RX in this context is from host to controller.

Reply:
`LINK_RX_POOL_SIZE` link structure used for all Controller to Host traffic, ACL DATA + Events, like BIG Complete and BIG Terminate in this case here. For Synchronized Receiver, it is Sync Established and Sync Lost event, hence the `* 2`.

Reply:
Ah. I though the `LINK` part of the name referred on-air. I get it now, thanks for the explanation.